mpqp/execution/devices.py

```diff
@@ -113,6 +127,34 @@ def is_gate_based(self) -> bool:
     def is_simulator(self) -> bool:
         return True
 
+    def is_noisy_simulator(self) -> bool:
```

Comment:
By definition all real devices are noisy, so we needed to define it only for simulators. One can still use the conjonction of is_simulator() and is_noisy() to check if the device is a noisy simulator, but this will made us call two functions, and require to define is_noisy so it returns also True for real devices

Reply:
I said real devices are noisy, meaning quantum computers. The question of being noisy or not is only relevant for simulators, that is why I think we should keep the restriction on the name

Reply:
Oh, right, I can't read x)
ok, you convinced me